src/tools/annotation/bidirectionalTool/moveHandle/getBaseData.js

```diff
@@ -0,0 +1,34 @@
+import external from '../../../../externalModules.js';
+import getDistance from './getDistance.js';
+
+export default function getBaseData(data, eventData, fixedPoint) {
```

Comment:
What is `data`?
Instead of passing in _all_ of `eventData`, can we scope this to:

```js
// optional
{
   columnPixelSpacing: int,
   rowPixelSpacing: int
}
```

What is the type/structure of `fixedPoint`?

Reply:
I renamed that to `toolData`.
Thank you!